src/themes/teams/components/Icon/svg/ProcessedIcons/icons-onenote-section.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react'
+import { TeamsSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16">
+      <path fill="none" d="M4 4h24v24H4z" />
+      <path
+        d="M19 25h-1v-1a.92.92 0 0 0-1-1h-2.5a2.57 2.57 0 0 1-2.5-2.5v-9A2.57 2.57 0 0 1 14.5 9H17a.92.92 0 0 0 1-1V7h1z"
+        fill-rule="evenodd"
```

Comment:
note, this should be defined as `fillRule`, no dash is allowed - similar to `border-color` vs `borderColor`. Currently this results in runtime error.